app/models/lettings_log.rb

```diff
@@ -435,13 +435,22 @@ def self.to_csv(user = nil)
     Csv::LettingsLogCsvService.new(user).to_csv
   end
 
+  def validation_beds
+    max_beds_in_rent_ranges = 4
+    if needstype == 2
+      0
+    else
+      beds.nil? ? nil : [beds, max_beds_in_rent_ranges].min
+    end
+  end
```

Comment:
```suggestion
  def beds_for_la_rent_range
    return 0 if is_supported_housing?
        
    beds.nil? ? nil : [beds, 4].min
  end
```

I think I'd re-write this slightly to improve readability. What do you think?

Reply:
That's better, I've used the MAX_BEDS value here too ofc :)